mast/templates/register.html

```diff
@@ -0,0 +1,40 @@
+{% extends "auth_template.html" %}
+{% block form %}
+<form class="form-signin">
+    <div class="text-center mb-4">
+        <h1 class="h2 mb-3">
+            Matfyz<br>
+            Activity<br>
+            <span class="primary_header">Sport<br>Tracker</span>
+        </h1>
+        <p>Compete, track and enjoy</p>
+    </div>
+
+    <div class="form-label-group">
+        <input type="text" id="inputUsername" class="form-control" placeholder="Username" required autofocus>
+        <label for="inputUsername">Username</label>
+    </div>
+
+    <div class="form-label-group">
+        <input type="email" id="inputEmail" class="form-control" placeholder="Email address" required autofocus>
+        <label for="inputEmail">Email address</label>
+    </div>
+
+    <div class="form-label-group">
+        <input type="password" id="inputPassword" class="form-control" placeholder="Password" required>
+        <label for="inputPassword">Password</label> 
+    </div>
+
+    <div class="form-label-group">
+        <input type="password" id="confirmPassword" class="form-control" placeholder="Confirm Password" required>
+        <label for="confirmPassword">Confirm Password</label> 
+    </div>
+
+    <div class="checkbox mb-3">
+        <!-- TODO: Add support for 3-rd party applications. -->
+        <a href="{{ url_for( 'login' ) }}">Already Signed-in?</a>
```

Comment:
"Signed in" = I successfully entered credentials and I was allowed to enter :D

Suggestion: "Already registered?" / "Do you already have an account? Go to Sign-in page"